Lombiq.Tests.UI/Services/UITestContext.cs

```diff
@@ -90,9 +90,9 @@ public class UITestContext
     public string TenantName { get; set; } = "Default";
 
     /// <summary>
-    /// Gets or sets the current Orchard Core admin prefix. When running UI tests on a site that
-    /// uses a custom admin prefix, this value should be set in the test. After that, navigation methods will be able
-    /// to use the custom URL.
+    /// Gets or sets the current Orchard Core admin prefix. When running UI tests on a site that uses a custom admin
+    /// prefix, this value should be set in the test.
+    /// After that, navigation methods will be able to use the custom URL.
```

Comment:
You needn't a line break before that (it can be one paragraph, but if you'd want multiple paragraphs, `<para>` is for that).